src/extensions/padding-controls/index.js

```diff
@@ -60,13 +61,23 @@ const paddingOptions = [
 const withPaddingControls = createHigherOrderComponent( ( BlockEdit ) => {
 	return ( props ) => {
 		const { name, attributes, setAttributes, isSelected } = props;
-		const { padding } = attributes;
+		const { padding, paddingCustom } = attributes;
+
+		const getMaxValue = () => {
+			const { maxValue = null } = props;
```

Comment:
The default `maxValue` should be passed through as a parameter instead of destructuring the props within here. The function should only have knowledge of the data passed into it.

Reply:
Good call JR. This was based on work done for the Gutter Controls component. That works differently however since its a class component as opposed to the functional component we am using for Padding Controls.
https://github.com/godaddy-wordpress/coblocks/blob/b3fae2ea8cf4fc4779d4c605bafbf877d815ec8e/src/components/gutter-control/gutter-control.js#L35-L43

This has been simplified and pushed back up.